client/package.json

```diff
@@ -8,16 +8,35 @@
     "@emotion/styled": "^11.10.4",
     "@mui/icons-material": "^5.10.9",
     "@mui/material": "^5.10.11",
-    "@reach/combobox": "^0.18.0",
     "@react-google-maps/api": "^2.13.1",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "antd": "^4.23.6",
+    "assert": "^2.0.0",
```

Comment:
damn thats a lot of dependencies can you check which ones aren't being used and delete them?